controller/Results.php

```diff
@@ -115,7 +154,7 @@ public function index()
         );
 
         $deliveryService = DeliveryAssemblyService::singleton();
-        $delivery = new core_kernel_classes_Resource(tao_helpers_Uri::decode($this->getRequestParameter('classUri')));
+        $delivery = new core_kernel_classes_Resource(tao_helpers_Uri::decode($this->getRequestParameter('uri')));
```

Comment:
Please use the parameter `id` without `tao_helpers_Uri::decode` as much as possible